lib/kdump_utils.pm

```diff
@@ -170,7 +170,8 @@ sub activate_kdump {
         $expect_restart_info = 1;
     }
     # ppcl64e and aarch64 needs increased kdump memory bsc#1161421
-    if (is_ppc64le || is_aarch64) {
+    # migration regression test cases need increase kdump memory since lot of services start
+    if (is_ppc64le || is_aarch64 || get_var('FLAVOR') =~ /Regression/) {
```

Comment:
Different system has different Physical memory allocations, however, in general, we 'd better set a rule to allocate kdump memory for this case.

For example, [not sure if it can work fine]
Physical mem > 4G 【kdump memory should not be less then 256M 】
Physical mem >2G and <4G 【kdump memory should not be less then 192M 】

Reply:
> If the condition is for migration regression job group, does the 'is_ppc64le || is_aarch64' still needed?
i suggest for regression group we increase memory for all platform.

Reply:
> Different system has different Physical memory allocations, however, in general, we 'd better set a rule to allocate kdump memory for this case.
> 
> For example, [not sure if it can work fine] Physical mem > 4G 【kdump memory should not be less then 256M 】 Physical mem >2G and <4G 【kdump memory should not be less then 192M 】

There is simple logic within current implementation:
my $memory_kdump = $memory_total >= 2048 ? 1024 : 320;

Reply:
That will be great, one more comments, I would suggest using at least 2Gb memory if we need cover kdump tests.
1Gb is too small to test this function

Reply:
> > If the condition is for migration regression job group, does the 'is_ppc64le || is_aarch64' still needed?
> > i suggest for regression group we increase memory for all platform.

I mean if the condition of  'is_ppc64le || is_aarch64'  added only for regression then could be removed since now the whole regression group included.

Reply:
'is_ppc64le || is_aarch64' is still needed for some cases